products/appengine/api.yaml

```
@@ -440,3 +440,39 @@ objects:
             name: 'shell'
             description: |
               The format should be a shell command that can be fed to bash -c.
+  - !ruby/object:Api::Resource
+    name: 'UrlDispatchRule'
```

```suggestion
    name: 'ApplicationUrlDispatchRules'
```

Just a nitpicky rename, I'd like to namespace this a little more + match what we're doing more closely. This will mean updating anywhere using the old name so you probably don't want to directly apply this suggestion using the UI.